app/_hub/_init/my-extension/_index.md

```diff
@@ -133,24 +133,38 @@ params: # Metadata about your plugin
   k8s_examples:
```

Comment:
Should this (and all boolean examples) be like this to match what we have for cloud, enterprise, and plus? 
```suggestion
  k8s_examples: # Boolean.
```